.github/workflows/lint.yml

```diff
@@ -0,0 +1,81 @@
+name: Lint
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  run-eslint:
+    name: Run eslint
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install Node.js dependencies
+        run: yarn
+        env:
+          SKIP_PREPARE: true
+
+      - name: Run eslint
+        uses: reviewdog/action-eslint@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          eslint_flags: 'src/'
+
+  run-stylelint-css:
+    name: Run stylelint (css)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install Node.js dependencies
+        run: yarn
+        env:
+          SKIP_PREPARE: true
+
+      - name: Run stylelint
+        uses: reviewdog/action-stylelint@v1
```

Comment:
You can also just run normal stylelint and use a problem matcher like [this](https://github.com/xt0rted/stylelint-problem-matcher)